chess/square.go

```diff
@@ -0,0 +1,150 @@
+package chess
+
+// A Square represents a square on the chess board.
+// Squares are ordered left-to-right, then bottom-to-top.
```

Comment:
nit: no hyphens needed

```suggestion
// Squares are ordered left to right, then bottom to top.
```